common/templates/layouts/base.njk

```diff
@@ -145,5 +145,6 @@
 {% endblock %}
 
 {% block bodyEnd %}
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
😬 How much does this bump up our page size?

Reply:
70k I would guess.

"Happy" to take another day rewriting the MoJ Design System to not rely on jquery. Unfortunately using a lighter-weight api-compliant alternative is not an option because of the heavy usage of `$.proxy